marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+
+bool   healthy
+string message
```

Comment:
I feel like that kind of status level is problematic because there isn't a concrete definition for what constitutes each error level, and different consumers of this message may have different definitions of what is an error versus warning.